antipatterns/no_legacy.json

```diff
@@ -0,0 +1,26 @@
+{
+  "antiPatterns": [
+    {
+      "name": "No Legacy",
+      "aliases": [
+        ""
+      ],
+      "description": "A strategy often preferred by large vendors where they do not want any obsolete parts in their new system.",
```

Comment:
Maybe one more sentence that legacy code is not always bad. If I only read the description it is not clear if it is bad or good.